src/survey.cpp

```diff
@@ -248,6 +248,10 @@ void SurveyConfiguration(nlohmann::json &survey)
 	}
 	if (BaseGraphics::GetUsedSet() != nullptr) {
 		survey["graphics_set"] = fmt::format("{}.{}", BaseGraphics::GetUsedSet()->name, BaseGraphics::GetUsedSet()->version);
+		const GRFConfig *extra_cfg = BaseGraphics::GetUsedSet()->GetExtraConfig();
+		if (extra_cfg != nullptr && extra_cfg->num_params > 0) {
+			survey["graphics_set_parameters"] = span<const uint32_t>(extra_cfg->param.data(), extra_cfg->num_params);
+		}
```

Comment:
To put Discord on GitHub: I think I prefer having this key always set, to make it more the same as how NewGRFs work. Makes analysing things just a bit easier (and always good to know things are deliberately empty)